sentences/sv/light_HassLightSet.yaml

```diff
@@ -13,9 +13,9 @@ intents:
           name: "all"
 
       - sentences:
-          - "(färga | ändra | byt) <name> [till] {color}"
+          - "(<change>) <name> [till] {color}"
```

Comment:
I think that we can add:
`kan du byta färg på sänglampan till rött`
Since `kan du` is added as skip_word it enables an additional sentence.

Reply:
As a test or in the sentence?

Reply:
I think that as test and adding sentence to match it. Will be some more options to the sentence.

Reply:
`(<change>)` should not have `()` around since it's already in the expansion_rules.

Reply:
> I think that we can add: `kan du byta färg på sänglampan till rött` Since `kan du` is added as skip_word it enables an additional sentence.

Can you add this separately?

> `(<change>)` should not have `()` around since it's already in the expansion_rules.

removed

Reply:
sure, lets add it as a new MR.